src/main/kotlin/org/eclipse/uprotocol/cloudevent/validate/CloudEventValidator.kt

```diff
@@ -114,12 +113,11 @@ abstract class CloudEventValidator {
         }
 
         override fun validateSink(cloudEvent: CloudEvent): ValidationResult {
-            val maybeSink: Optional<String> = UCloudEvent.getSink(cloudEvent)
-            if (maybeSink.isPresent) {
-                val sink: String = maybeSink.get()
+            var result: ValidationResult = ValidationResult.success()
```

Comment:
Could we remove the mutable state `result` here or would that become unreadable?
Maybe multiple returns is better in this situation (similar to `Notification::validateSink`)?